deploy/chart/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v2
 name: spiceai
 description: Spice.ai OSS
-version: 0.2.3
+version: 0.2.4
```

Comment:
I think we bumped this at the end of last release, might want to check.

Reply:
![CleanShot 2025-01-14 at 15 03 51@2x](https://github.com/user-attachments/assets/11e423e5-04a2-4b8d-a6ec-8e5a5619ac56)
https://github.com/spiceai/spiceai/pull/4090/commits/1b61bfd224e84ebe0f87cd3023d7708b47f5bcc0

chart 0.2.3 was updated together with rc.4